mucommander-translator/src/main/resources/dictionary_ko.properties

```diff
@@ -200,15 +204,13 @@ SwapFolders.tooltip = 왼쪽 및 오른쪽 폴더 교체
 SetSameFolder.label = 같은 폴더로 설정
 SetSameFolder.tooltip = 왼쪽 및 오른쪽 패널을 같은 디렉토리로 설정
 NewWindow.label = 새 창
-NewWindow.tooltip = 새 창으로 열기
+NewWindow.tooltip = 새 창 열기
 Open.label = 열기
-Open.tooltip = 폴더 열기 / 압축파일 열기 / 실행
-OpenCommandPrompt.label = 명령 프롬프트 열기
-OpenCommandPrompt.tooltip = 명령 프롬프트를 열고 현재 폴더로 변경
-OpenNatively.label = 기본적으로 열기
-OpenNatively.tooltip = 시스템의 파일 연결로 선택한 파일 실행
+Open.tooltip = 폴더 입력 / 보관 입력 / 실행
+OpenNatively.label = 기본으로 열기
+OpenNatively.tooltip = 선택한 파일을 시스템의 파일 연결과 함께 실행
 OpenInNewTab.label = 새 탭에서 열기
-ShowInEnclosingFolder.label = 동봉된 폴더에 표시
+ShowInEnclosingFolder.label = 동봉 폴더에 표시
```

Comment:
@VenusGirl  does it mean something like parent folder? if not, maybe we should replace the term 'enclosing' with 'parent'